java/code/src/com/suse/manager/utils/SaltUtils.java

```diff
@@ -1481,7 +1481,8 @@ private static void updatePackages(MinionServer server,
                 )
                 .collect(Collectors.toMap(
                         SaltUtils::packageToKey,
-                        Function.identity()
+                        Function.identity(),
+                        (first, duplicate) -> first // Deal with duplicates: ignore additional entries
```

Comment:
@cbbayburt as the install date is something we use sometimes it would be good to keep the entry with the latest install date.